gatorgrade/input/in_file_path.py

```diff
@@ -7,3 +7,33 @@ def parse_yaml_file(file_path):
     with open(file_path, encoding="utf8") as file:
         data = yaml.load_all(file, Loader=yaml.FullLoader)
         return list(data)
+
+
+def reformat_yaml_data(data):
+    """Reformat the raw data from a YAML file into a list of tuples."""
+    reformatted_data = []
+    if len(data) == 2:
+        data.pop(0)  # Removes the setup commands
```

Comment:
Hey Jacob, your code looks great so far. There were no linting errors. However, are we not going to need the shell setup commands? Or are you going to use the setup shell commands data and pass it in your next commit or next PR?

Reply:
The shell commands will be acquired before this function is run, so in between running the `parse_yaml_file` function and the `reformat_yaml_data` function, we can get the data from the output of `parse_yaml_file` by checking if there are 2 values in the list, and if there are, the first value will be the setup commands. Those commands are stored as a dictionary with the key being `"setup"` and the value being a singular string of each setup command. The different commands for setup are separated in the string by `\n`, which represents a new line. Let me know if you need any more info about that!